src/importer/Program.cs

```diff
@@ -11,36 +12,50 @@ static void Main(string[] args)
         {
             var app = new CommandLineApplication();
             var folderOption = app.Option("-$|-f|--folder <folder>", "Folder to read UCAS .xls files from ", CommandOptionType.SingleValue);
+            var urlOption = app.Option("-u|--url|--api-url <url>", "URL of the ManageCourses API", CommandOptionType.SingleValue);
+            var apiKeyOption = app.Option("-k|--key|--api-key <key>", "Admin Key for the ManageCourses API", CommandOptionType.SingleValue);
+            
             app.HelpOption("-?|-h|--help");
             app.Execute(args);
 
-            var users = new CsvReader().ReadUsers(folderOption.Value());
-            var organisations = new CsvReader().ReadOrganisations(folderOption.Value());
-            var organisationInstitutions = new CsvReader().ReadOrganisationInstitutions(folderOption.Value(), organisations);
-            var organisationUsers = new CsvReader().ReadOrganisationUsers(folderOption.Value(), organisations, users);
+            if (!folderOption.HasValue()) {
+                Console.WriteLine("Missing --folder option");
+                return;
+            }
+
+            if (!urlOption.HasValue()) {
+                Console.WriteLine("Missing --url option");
+                return;
+            }
+
+            if (!apiKeyOption.HasValue()) {
+                Console.WriteLine("Missing --key option. Attempting import without key...");
+            }
```

Comment:
Why did you miss out return?

api server not going to accept blanks

Reply:
that's fair - correcting